test_outputs/pub_module_level_const_missing.output.ron

```diff
@@ -0,0 +1,118 @@
+{
+    "./test_crates/pub_module_level_const_missing/": [
+        {
+            "name": String("pub_const_in_global_will_remove"),
+            "span_begin_line": Uint64(3),
+            "span_filename": String("src/lib.rs"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("pub_const_in_global_will_rename"),
+            "span_begin_line": Uint64(4),
+            "span_filename": String("src/lib.rs"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("pub_const_in_global_will_be_static"),
+            "span_begin_line": Uint64(5),
+            "span_filename": String("src/lib.rs"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("pub_const_in_global_will_be_private_const"),
+            "span_begin_line": Uint64(6),
+            "span_filename": String("src/lib.rs"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("pub_const_in_global_will_be_private_static"),
+            "span_begin_line": Uint64(7),
+            "span_filename": String("src/lib.rs"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("pub_const_in_module_will_remove"),
+            "span_begin_line": Uint64(11),
+            "span_filename": String("src/lib.rs"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("pub_const_in_module_will_rename"),
+            "span_begin_line": Uint64(12),
+            "span_filename": String("src/lib.rs"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("pub_const_in_module_will_be_static"),
+            "span_begin_line": Uint64(13),
+            "span_filename": String("src/lib.rs"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("pub_const_in_module_will_be_private_const"),
+            "span_begin_line": Uint64(14),
+            "span_filename": String("src/lib.rs"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("pub_const_in_module_will_be_private_static"),
+            "span_begin_line": Uint64(15),
+            "span_filename": String("src/lib.rs"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("pub_const_in_module_will_re_export"),
```

Comment:
This being marked as a missing pub const is unexpected (since my understanding is that we are re-exporting). For reference:

**Current**
```rust
mod re_exporter {
    pub const special_const: i32 = 0;
    pub static special_static: i32 = 0;
}
pub mod my_module {    
    pub use crate::re_exporter::special_const as 
        pub_const_in_module_will_re_export;
}
```
**Baseline**
```rust
pub mod my_module {
    pub const pub_const_in_module_will_re_export: i32 = 0;
}
```

Reply:
After some more digging, it appears that rust exports don't inherit the static/const-ness of the original variable and must be declared separately... will update tests to reflect that

Reply:
Oh, interesting! Would love to learn more about the experiment you ran to find this, because this seems unexpected to me and I'm a bit worried it might be a bug in my code.

Reply:
Sorry, I think I was mistaken. I was under the impression that `pub const use` or `pub static use` was something that existed in rust after reading some stackoverflow answers. After some testing, it seems that rust does inherit the static/const-ness of the original variable, though some confirmation would be nice since I can't get the compiler to spit out whether something is a static or const and trying to assign to a re-exported const doesn't tell me much since doing so is invalid in the first place.

Reply:
Based on this test case I just added, it seems that `pub use` works correctly and normally over both `const` and `static` items :+1:

I'm not sure why the test cases here are failing, but I'm happy to take a deeper look if you'd like me to.